src/modules/TextInput/ErrorTextInput.tsx

```diff
@@ -36,9 +36,9 @@ export const ErrorTextInput: React.FC<IProps & WithStylesProps> = ({
         {...props}
         hasError={error !== null && error !== undefined}
       />
-      {error && typeof error === "string" && (
-        <div className={cx(styles.errorLabel)}>{error}</div>
-      )}
+      <div className={cx(styles.errorLabel)}>
```

Comment:
Is this change intended? Do we want to have the error label element even without an error?

Reply:
@nunoliveira12

Reply:
To keep the space of the error so it doesnt jump. There are alternatives to this